src/runtime/nuxt-image-mixins.ts

```diff
@@ -78,6 +78,11 @@ export default {
   },
   async fetch () {
     await this.fetchMeta()
+
+    if (this.$nuxt.context.ssrContext && this.$nuxt.context.ssrContext.isGenerating) {
+      // eslint-disable-next-line no-unused-expressions
+      this.sizes
```

Comment:
??

Reply:
@pi0  Oh, I forgot to put a comment above this.
In the `nuxt generate` we need to generate image srcsets and download them into `dist` directory, but because of lazy loading, the sizes does not calculate on the server side. With this we make sure that sizes are generates and downloads on static build.